plugins/base/src/main/kotlin/transformers/pages/sourcelinks/SourceLinksTransformer.kt

```diff
@@ -24,36 +24,47 @@ import java.io.File
 
 class SourceLinksTransformer(val context: DokkaContext) : PageTransformer {
 
-    private val builder : PageContentBuilder =  PageContentBuilder(
+    private val builder : PageContentBuilder = PageContentBuilder(
         context.plugin<DokkaBase>().querySingle { commentsToContentConverter },
         context.plugin<DokkaBase>().querySingle { signatureProvider },
         context.logger
     )
 
-    override fun invoke(input: RootPageNode) =
-        input.transformContentPagesTree { node ->
+    override fun invoke(input: RootPageNode): RootPageNode {
```

Comment:
A bit of refactoring to make it not run if sources are not configured and to not parse configuration on every documentable